packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -41,7 +40,6 @@ const DEFAULT_MINIMUM_FONT_SIZE_LIMIT = '14px';
  * @param {?string}       args.minimumFontSize       Minimum font size for any clamp() calculation. Optional.
  * @param {?number}       args.scaleFactor           A scale factor to determine how fast a font scales within boundaries. Optional.
  * @param {?number}       args.minimumFontSizeFactor How much to scale defaultFontSize by to derive minimumFontSize. Optional.
- * @param {?number}       args.maximumFontSizeFactor How much to scale defaultFontSize by to derive maximumFontSize. Optional.
```

Comment:
Just double-checking I'm following: we're removing this param because the selected font size will now be treated as "max" if no maximum is provided... that sounds good to me. It looks like we also need to update the comment around line 19 or so that references this param.

Reply:
Good spotting. Updating...